src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -88,27 +131,22 @@ protected static ConfigDef baseConfigDef() {
       int order = 0;
       configDef
           .define(TYPE_NAME_CONFIG, Type.STRING, Importance.HIGH,
-                  "The Elasticsearch type name to use when indexing.",
+                  TYPE_NAME_DOC,
                   group, ++order, Width.SHORT, "Type Name")
```

Comment:
Also a bit surprised that checkstyle doesn't require one argument per line. Maybe we turned this off.